jest.config.cjs

```diff
@@ -225,12 +225,12 @@ const customJestConfig = {
       statements: 68,
     },
     "src/cloud-functions/index.js": {
-      branches: 63,
+      branches: 60,
```

Comment:
The annoying thing here is that I don't know why this coverage went down, given that I did not actually touch that file :/ If anyone has any thoughts on that, let me know.

Reply:
We can always open up the `index.html` in coverage folder to see exactly which branches are no longer being tested. But it's probably due to changes to underlying functions (which I don't see any connections between this file and `index.js`.. weird) that file depends on

Reply:
Unfortunately I see no changes when eyeballing it, other than the actual number:

![Screenshot 2023-08-18 at 16-23-12 Code coverage report for src_cloud-functions_index js](https://github.com/mozilla/blurts-server/assets/4251/ac336005-176d-4263-a27b-646a93b66f42)
![Screenshot 2023-08-18 at 16-23-23 Code coverage report for src_cloud-functions_index js](https://github.com/mozilla/blurts-server/assets/4251/12f9d908-4e2a-457c-8fe4-9a3afa77ae88)